server/config/configresource/config_resource.go

```diff
@@ -32,18 +32,32 @@ type repository struct {
 	db *sql.DB
 }
 
+func (r *repository) Current(ctx context.Context) Config {
+	defaultConfig := Config{
+		AnalyticsEnabled: true,
+	}
+	list, err := r.List(ctx, 1, 0, "", "", "")
+	if err != nil || len(list) != 1 {
+		// TODO: log error
+		return defaultConfig
+	}
+
+	return list[0]
```

Comment:
in case of multiple records in the DB you are using the first one, right?

Reply:
If that will be fixed in another PR, this one looks good to me